.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  APPLICATION_ENV: 'development'
+  APPLICATION_STORE: 'DE'
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.4'
```

Comment:
Same issue
min/max needs to be properly addressed

You can also refer to decimal object or the other already merged ones as the main library on this.